packages/uploadthing/src/sdk/index.ts

```diff
@@ -257,7 +257,7 @@ export class UTApi {
     return json.files;
   }
 
-  async renameFile(
+  async renameFiles(
```

Comment:
Shouldn't we also change the `/api/renameFile` to `/api/renameFiles` on line **276**?